server/src/main/java/com/testsigma/model/TestDataType.java

```diff
@@ -15,7 +15,7 @@ public enum TestDataType {
   raw(1, "raw"),
   parameter(2, "parameter"),
   runtime(3, "runtime"),
-  environment(4, "environment"),
+  global(4, "global"),
```

Comment:
@manikanta-reddy-t You also need to migrate the existing data in test step table from environment to global

Reply:
written migration query for this